src/uct/ib/dc/base/dc_iface.c

```diff
@@ -219,11 +219,27 @@ void uct_dc_iface_set_quota(uct_dc_iface_t *iface, uct_dc_iface_config_t *config
                                 ucs_min(iface->super.config.tx_qp_len, config->quota);
 }
 
+static uint8_t uct_dc_version(uct_ib_device_t *dev)
```

Comment:
IMHO better rename to uct_ib_device_get_dc_version(), and move to ib_device.c

Reply:
it would cause dependency of base module from dc, reworked a bit different way